libraries/USB/examples/FirmwareMSC/FirmwareMSC.ino

```diff
@@ -9,13 +9,7 @@ void loop(){}
 #if !ARDUINO_USB_MSC_ON_BOOT
 FirmwareMSC MSC_Update;
 #endif
-#if ARDUINO_USB_CDC_ON_BOOT
 #define HWSerial Serial0
```

Comment:
you no longer need this define. Change the rest of the example code to call `Serial0` for UART and `USBSerial` for CDC. Same for the other applicable examples.

Reply:
That's something I was thinking about yesterday as well... 
I left it there because the user could change it to any other Print communication channel...
What do you say? We can change it to "PrintObj" instead of "HWSerial"

Reply:
Looking at the code, I think that with those changes, it needs to just call Serial and not even check this define. If it's set to be UART, it will go there, else whatever CDC it's set to.

Reply:
Replace all HWSerial with Serial and delete references to USBSerial

Reply:
OK.